src/app/overview/container.tsx

```diff
@@ -32,11 +30,14 @@ export default class Overview extends React.Component<Props & DispatchProps, any
   public componentDidMount(): void {
     this.props.getCurrentPriceInUSD()
     this.props.getCurrentPriceInBTC()
-    this.props.getTransactions()
   }
 
   public render(): any {
     const values: any = mapValues(this.props.overview, (value: string) => parseFloat(value).toFixed(MAX_DECIMALS))
+    const transactions: any = this.props.transactions.transactions ?
```

Comment:
Can we create a `const transactions = this.props.transactions.transactions` ? This is too big

Reply:
Also, see below comment:
"Do we need this `|| []`? The default state on the reducer is the empty array already right?"

Reply:
no it was returning an error